src/components/Preferences/BodyApplication.vue

```diff
@@ -20,7 +24,13 @@ export default Vue.extend({
     }
   },
   data() {
-    return { activeTab: 'environment' };
+    return { activeTab: 'behavior' };
+  },
+  computed: {
+    ...mapGetters('preferences', ['isPlatformWindows']),
+    getActiveTab(): string {
```

Comment:
We actually no longer need `getActiveTab` either; when the tab bar is hidden, it's not possible to switch tabs (even with the keyboard).